pyecharts/template.py

```diff
@@ -8,8 +8,29 @@
 import codecs
 from jinja2 import Environment, FileSystemLoader
 import pyecharts.constants as constants
+import json
 
 
+def get_resource_dir(folder):
+    """
+
+    :param folder:
+    :return:
+    """
+    current_path = os.path.dirname(__file__)
+    resource_path = os.path.join(current_path, folder)
+    return resource_path
+
+
+#with codecs.open(os.path.join(get_resource_dir('templates'), 'js', 'echarts', 'registry.json'), 'r', 'utf-8') as f:
+with open(os.path.join(get_resource_dir('templates'), 'js', 'echarts', 'registry.json'), 'rb') as f:
+    content = f.read().decode('utf-8')
+    CONFIG = json.loads(content)
+
+DEFAULT_JS_LIBRARIES = CONFIG['FILE_MAP']
```

Comment:
They are loaded here.

Reply:
got it, it will be unified management by sub module.

Reply:
yes. jupyter-echarts expose all its js libraries via 'registry.json' and will do so in the future. i.e. map files from other countries, or other echarts extensions